scripts/Muon/GUI/Common/pairing_table_widget/pairing_table_widget_presenter.py

```diff
@@ -51,10 +51,17 @@ def handle_guess_alpha_clicked(self, row):
         self.guessAlphaNotifier.notify_subscribers([pair_name, group1, group2])
 
     def handle_data_change(self, row, col):
-        changed_item = self._view.get_table_item_text(row, col)
+        table = self._view.get_table_contents()
+        changed_item = table[row][col]
         update_model = True
         if col == 0 and not self.validate_pair_name(changed_item):
             update_model = False
+        if col == 1:
```

Comment:
can we have some constant for these numbers to make it clearer what these numbers mean?